src/game/boe.actions.cpp

```diff
@@ -2280,6 +2280,21 @@ void do_save(short mode) {
 	redraw_screen(REFRESH_TEXT);
 }
 
+void do_abort() {
+	if(recording){
+		record_action("do_abort", "");
+	}
+	if(overall_mode != MODE_STARTUP) {
```

Comment:
Isn't this wrong? Shouldn't it be checking `party_in_memory`? If you have a party open at the startup screen and choose abort, I think it should confirm, right?

Though I guess this is exactly what it does right now, so that would be a separate issue…

Reply:
I agree the condition of the if should be `party_in_memory`, not what it is.

Other places where quitting, there's no check to skip confirming if a save just happened. So I don't know how to easily implement that.

But changing the `if` condition, is trivial so I'm happy to put that in this PR.